mon-pix/app/models/user-oidc-authentication-request.js

```diff
@@ -3,23 +3,41 @@ import { memberAction } from 'ember-api-actions';
 
 export default class UserOidcAuthenticationRequest extends Model {
   @attr('string') email;
+  @attr('string') username;
   @attr('string') password;
   @attr('string') identityProvider;
   @attr('string') authenticationKey;
   @attr('string') accessToken;
   @attr('string') logoutUrlUUID;
+  @attr('string') fullNameFromPix;
+  @attr('string') fullNameFromExternalIdentityProvider;
+  @attr() authenticationMethods;
 
   login = memberAction({
-    path: 'token-reconciliation',
+    path: 'check-reconciliation',
     type: 'post',
     before() {
       const payload = this.serialize();
       delete payload.data.attributes['access-token'];
       delete payload.data.attributes['logout-url-uuid'];
+      delete payload.data.attributes['username'];
+      delete payload.data.attributes['full-name-from-pix'];
+      delete payload.data.attributes['full-name-from-external-identity-provider'];
+      delete payload.data.attributes['authentication-methods'];
       return payload;
     },
     after(response) {
-      return response?.data?.attributes;
+      if (response.data?.attributes) {
+        return {
+          fullNameFromPix: response.data.attributes['full-name-from-pix'],
+          fullNameFromExternalIdentityProvider: response.data.attributes['full-name-from-external-identity-provider'],
+          email: response.data.attributes['email'],
+          username: response.data.attributes['username'],
+          authenticationMethods: response.data.attributes['authentication-methods'],
+        };
+      } else {
+        return response;
+      }
```

Comment:
le else n'est pas necessaire

```suggestion
      }
      
      return response;
```